packages/salesforcedx-vscode-core/src/commands/dataQuery.ts

```diff
@@ -303,7 +274,7 @@ export const escapeCSVField = (field: string): string => {
 /**
  * Formats a field value for CSV export
  */
-export const formatFieldValue = (value: any): string => {
+export const formatFieldValue = (value: unknown): string => {
```

Comment:
`unknown` lets you narrow the type (ex: not null, not undefined, is object, etc).  That'll help TS make sure we didn't miss any possibilities.

`any` just stays super wide no matter what.